### ISSUE TITLE
Issue testing swagger service using curl command on docker container

### ISSUE THREAD

**Issue by spathan1985** [cloudmesh-community/hid-sp18-516#21]
Hello @laszewsk Professor, TAs, 
For the swagger assignment, I'm able to start the docker container but not able to test to it out using curl commands. It gives the following error:

```
ENV2) spathan@spathan-VirtualBox:~/github/cloudmesh-community/hid-sp18-516/swagger/cloudmesh/localhost$ curl -H "Authorization: Basic YWRtaW46c2VjcmV0" -H "Con
tent-Type:application/json" -X POST -d '{"userName": "spathan", "hostName": "localhost", "command": "ls -l"}' http://localhost:8080/cloudmesh/localhost
curl: (7) Failed to connect to localhost port 8080: Connection refused
(ENV2) spathan@spathan-VirtualBox:~/github/cloudmesh-community/hid-sp18-516/swagger/cloudmesh/localhost$ 
(ENV2) spathan@spathan-VirtualBox:~/github/cloudmesh-community/hid-sp18-516/swagger/cloudmesh/localhost$ 
(ENV2) spathan@spathan-VirtualBox:~/github/cloudmesh-community/hid-sp18-516/swagger/cloudmesh/localhost$ curl -i http://127.0.0.1:8080/cloudmesh/localhost
curl: (7) Failed to connect to 127.0.0.1 port 8080: Connection refused
(ENV2) spathan@spathan-VirtualBox
```

The swagger_server is running and container is running as well:
```
(ENV2) spathan@spathan-VirtualBox:~/github/cloudmesh-community/hid-sp18-516/swagger/cloudmesh/localhost$ sudo docker run --expose 8080 cloudmesh-localhost
cd server/ && python -m swagger_server
 * Running on http://127.0.0.1:8080/ (Press CTRL+C to quit)
```

```
(ENV2) spathan@spathan-VirtualBox:~/github/cloudmesh-community/hid-sp18-516/swagger/cloudmesh/localhost$ sudo docker container ls
[sudo] password for spathan: 
CONTAINER ID        IMAGE                 COMMAND             CREATED             STATUS              PORTS               NAMES
7f9ada331f3b        cloudmesh-localhost   "make start"        27 minutes ago      Up 27 minutes       8080/tcp            jovial_brattain
```

Request you to please take a look. Github has the latest code/files, more information can be found on README.md.

Thanks & regards,
Shagufta

**Comment by spathan1985** [comment]
@laszewsk Professor, after I spoke to you last, I just had given 'Hello World' string to work on the container and it did. Then I only gave the 'command' in the subprocess_checkoutput command (left out the ssh part from the command) and this worked too. So I realized that it is some problem with doing SSH on the container. For now the I'm able to execute the specified command on the container. 

Updated README.md with the latest details. Hence closing this issue. 
Thanks for your help.
